yad: add page (#20753)
tldr-pages/tldr: pages/linux/yad.md

# yad

> > Display GTK+ dialogs from shell scripts.

Reviewer: ```suggestion
> Display GTK+ dialogs from shell scripts.
> See also: `zenity`.
```

Oh, and since yad is a fork of [zenity](https://github.com/ncruces/zenity), I think it would be good to add a _see also_ mentioning it.

> More information: <https://manned.org/man/yad>.

- Display a text information dialog:

> `yad --text-info --filename={{path/to/file.txt}}`

Reviewer: ```suggestion
- Display the contents of a file in a text information dialog:

`yad --text-info --filename "{{path/to/file}}"`
```

Does it only accept .txt files?

- Prompt for text input:

> `yad --entry --text={{Enter your name:}}`

Reviewer: ```suggestion
- Open a text entry dialog and return the input to `stdout`:

`yad --entry --text "{{message}}"`
```

- Select a file:

> `yad --file --title={{Select a file}}`

Reviewer: ```suggestion
- Open a file picker with a specific title:

`yad --file --title "{{title_message}}"`
```

- Select a date from a calendar:

> `yad --calendar --title={{Select a date}}`

Reviewer: ```suggestion
- Open a date picker dialog with a specific title:

`yad --calendar --title "{{title_message}}"`
```

- Display a list:

> `yad --list --column={{Name}} --column={{Age}} {{Alice}} {{20}} {{Bob}} {{30}}`

Reviewer: ```suggestion
- Display a list dialog with multiple columns and data:

`yad --list --column "{{col1}}" --column "{{col2}}" {{"col1_row1" "col2_row1" "col1_row2" "col2_row2" ...}}`
```

Here is my idea to make the command example more generalized, but maybe could be further improved...

# yad

> Display GTK+ dialogs from shell scripts.

> > More information: <https://github.com/v1cont/yad>.

Reviewer: ```suggestion
> More information: <https://manned.org/man/yad>.
```

there isn't really much info about the commands in their repo

- Display an information dialog:

`yad --info --text={{Hello}}`

- Ask a yes/no question:

> `yad --question --text={{Continue?}}`

Reviewer: cant find `--question` flag as well
